src/rt-app.c

```diff
@@ -927,7 +927,6 @@ static void set_thread_priority(thread_data_t *data, sched_data_t *sched_data)
 				if (ret != 0) {
 					log_critical("[%d] setpriority "
 					     "returned %d", data->ind, ret);
-					errno = ret;
```

Comment:
I think we should never be setting errno directly when detecting errors from library calls so LGTM.